config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller:latest
+      - image: gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller:release-0.4
```

Comment:
We're not yet building images on postsubmit for the `release-0.4` branch yet.. @vincepri

Reply:
We'll have to enable it in `test-infra` first, if you wanna do that @tahsinrahman

Reply:
I don’t have any experience  on test-infra, can you give me some references? Thanks!

Reply:
https://github.com/kubernetes/test-infra/pull/14482/files

Reply:
Thanks!
Is it ok? https://github.com/kubernetes/test-infra/pull/14744